src/modules/hooks/useBlockExplorer/useBlockExplorer.test.ts

```diff
@@ -0,0 +1,28 @@
+import { renderHook } from '@testing-library/react';
+import * as wagmi from 'wagmi';
+import { mainnet, polygon } from 'wagmi/chains';
+import { useBlockExplorer } from './useBlockExplorer';
+
+describe('useBlockExplorer', () => {
```

Comment:
Can we add two more tests to verify that the hook:
1. Uses the first chain set on the wagmi provider when chainId property is not set
2. Uses the first chain passed as parameters when chain-id is not defined

Reply:
✅ https://github.com/aragon/ods/pull/220/commits/d917b9ea40962ded12c811ec0c207f7e5ca65be8